ee/vellum_ee/workflows/display/workflows/base_workflow_display.py

```diff
@@ -892,9 +893,26 @@ def serialize_module(
         if additional_files:
             exec_config["module_data"] = {"additional_files": cast(JsonObject, additional_files)}
 
+        dataset = None
```

Comment:
let's be sure to add one test near our other `serialize_module` tests to show dataset support